Library/Homebrew/cmd/tap-info.rb

```diff
@@ -64,6 +64,7 @@ def print_tap_info(taps)
             puts unless i.zero?
             info = "#{tap}: "
             if tap.installed?
+              info += "Installed"
```

Comment:
This needs a whitespace after `Installed`, but the terminology here is a bit misleading as we're showing the number of items in the tap, not the number of items that are installed.

Reply:
"Available" could be better

Reply:
@razvanazamfirei What kind of whitespace do you mean? The original string value appended didn't have trailing whitespace.

@SMillerDev The original code checks `tap.installed?`, and the else block uses "Not installed", so wouldn't "Installed" be best to match?